e2e/utils.go

```diff
@@ -162,9 +162,18 @@ func injectString(original, injection, after string) string {
 
 func k8sVersion() string {
 	// split the version string (like 'v1.29.6+k0s.0') into the k8s version and the k0s revision
-	verParts := strings.Split(os.Getenv("K0S_VERSION"), "+")
+	verParts := strings.Split(os.Getenv("EXPECT_K0S_VERSION"), "+")
```

Comment:
this wasnt actually working since actions have a limit of 10 inputs. it was just using the default from the makefile. i've changed this variable name to avoid the collision.

Reply:
That would do it